components/cards/challenge/_partials/Course.vue

```diff
@@ -0,0 +1,52 @@
+<template>
```

Comment:
I would move this component inside `components/cards/challenge` and name it something like `Teaser` or `Overview`.